src/request/import/Import.css

```diff
@@ -70,3 +70,14 @@ button.go-to-words {
     margin: 4rem;
     margin-right: 0;
 }
+
+.page#choose-key-type button:not([disabled]),
+.page#words button:not([disabled]) {
+    background: #724ceb;
+    color: white;
```

Comment:
Styling of disabled buttons needs to be fixed in the future when we have an official style for them.